.circleci/config.yml

```diff
@@ -85,7 +85,7 @@ jobs:
       - run: go build -i ./pilot/test/integration
       - run: JSONPATH='{range .items[*]}{@.metadata.name}:{range @.status.conditions[*]}{@.type}={@.status};{end}{end}'; until sudo kubectl get nodes -o jsonpath="$JSONPATH" 2>&1 | grep -q "Ready=True"; do sleep 1; done
       - run: sudo -E kubectl cluster-info
-      - run: make setup
+      - run: make kubelink setup
```

Comment:
do you need the setup here? All it does is to create the symlink.

Reply:
Ah, good point. For some reason, I thought that the setup target was doing more than that. I'll fix that up.